### ISSUE TITLE
Invalid SOH character written to image-description

### ISSUE THREAD

**Issue by blasky** [kleisauke/net-vips#144]
I have been writing OME-Tiff files for a while now with NetVips. My xmlString is validated before I write it. And most of the time everything is fine.
```
omeImage = omeImage.Mutate(
   x => { x.Set(GValue.GStrType, "image-description", xmlString); }
);

omeImage.Tiffsave( [with various options] );
```
But intermittently, the OME-Tiff is written and then fails to validate.
```
org.xml.sax.SAXParseException: Content is not allowed in trailing section.
```
Looking at the OME-Xml extracted from the Tiff when it fails, there is indeed an illegal Start Of Header (SOH) character at the end of the image-description as written.
```
<OME>
...
</OME>

```
Does this ring any bells? Is there anything I should be doing in the Mutate() or Tiffsave() calls to protect the integrity of the Tiff metadata? The invalid character only shows up every now and then, but when it does it means we have produced an invalid OME-Xml that will not validate against the schema.

**Comment by kleisauke** [comment]
I'll close this issue for now, without a reproduction it'll be hard to debug further. Please re-open if you are able to provide the (standalone) code that exhibit this behaviour.

**Comment by kleisauke** [comment]
Now that I've looked at this issue again, which version of NetVips was this tested on? It could possibly be a duplicate of #131, which was fixed in NetVips v2.0.1, but I think that issue only affected Mono.
